test/unit/validators/email_validator_test.rb

```diff
@@ -30,4 +31,10 @@ def setup
   test 'should allow blank' do
     assert @validatable.valid?
   end
+
+  test "user's email address can be UTF-8 encoded" do
+    user = FactoryGirl.build(:user, :with_utf8_mail)
```

Comment:
Use `@validatable.mail` when in this file, see the above examples - it's not testing the user model, it's testing the validator directly (which was the point in moving it, this is more precise and nothing changed in the user model).